homeworks/vlad.makedon_Vladmaked/homework_6/user-info.js

```diff
@@ -0,0 +1,27 @@
+const user = {};
+let fullName;
+let phone;
```

Comment:
please avoid interacting with global scope in this case, best solution is to create a wrapper function for what you are trying to do

Reply:
done